### ISSUE TITLE
[Failing Test] [sig-node] startup_probe_test.go

### ISSUE THREAD

**Issue by zouyee** [kubernetes/kubernetes#82747]
**Which jobs are failing**:
ci-kubernetes-node-kubelet-serial

**Which test(s) are failing**:
StartupProbe [Serial] [Disruptive] [NodeFeature:StartupProbe]

**Since when has it been failing**:

**Testgrid link**:
https://testgrid.k8s.io/sig-node-kubelet#node-kubelet-serial

**Reason for failure**:
```
It's not enough to set this flag in the kubelet because the apiserver needs it too
```
**Anything else we need to know**:

**Comment by zouyee** [comment]
@kubernetes/sig-node-test-failures

**Comment by odinuge** [comment]
Tests are no longer running in CI, but tests are still failing. 

One failure is related to https://github.com/kubernetes/kubernetes/issues/84178 and another one is related to https://github.com/kubernetes/kubernetes/pull/84179.

/reopen

**Comment by odinuge** [comment]
Test failures can now be tracked in https://testgrid.k8s.io/sig-node-kubelet#node-kubelet-serial-alpha

**Comment by spiffxp** [comment]
/close
It looks like this was solved around Nov 13-14, which is when #84279 merged

https://storage.googleapis.com/k8s-gubernator/triage/index.html?date=2019-11-20&pr=1&test=StartupProbe
